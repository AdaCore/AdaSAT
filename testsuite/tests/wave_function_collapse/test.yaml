project_file: prj.gpr
executable: wfc
