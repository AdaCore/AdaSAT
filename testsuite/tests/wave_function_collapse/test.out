Solved
         
         
         
         
        ^
 ^    ^ ^
^^  ^ ^^^
0xxx0x000
_________

